tests/e2e/utils/KubernetesCommandLineToolsExecutor.ts

```diff
@@ -115,6 +115,12 @@ export class KubernetesCommandLineToolsExecutor extends ShellExecutor {
         this.execWithLog(`${this.KUBERNETES_COMMAND_LINE_TOOL} delete project ${projectName} -n ${this.namespace} && sleep 5s`);
     }
 
+    private isUserLoggedIn(): boolean {
+        const whoamiCommandOutput: ShellString = this.execWithLog('oc whoami && oc whoami --show-server=true');
```

Comment:
It's interesting to know what is the reason to call `oc whoami` command twice at a time?

Reply:
One shows username and second - server url

Reply:
Got it. Thanks!